ui/app/manager/src/index.ts

```diff
@@ -233,6 +233,18 @@ fetch(configURL).then(async (result) => {
             });
         }
 
+        // When the page is not present in the header, move the PageProvider to the end of the array.
+        // This is to prevent the landing page for the user not being visible in the header
+        const realmAppConfig = orAppConfig.realms[manager.displayRealm] || orAppConfig.realms.default;
+        if(realmAppConfig) {
+            const headerPaths = [...realmAppConfig.header.mainMenu, ...realmAppConfig.header.secondaryMenu].map(item => item.href);
+            pages.forEach(pageProvider => {
+                if(!headerPaths.includes(pageProvider.name)) {
+                    pages.push(pages.splice(pages.indexOf(pageProvider), 1)[0]);
+                }
+            })
```

Comment:
My JS is getting rusty but modifying the collection whilst iterating...does that even work in JS? Maybe a quirk of how arrays are managed allows this.

Reply:
Good point. Adjusted it to use `.filter()` instead.